app/views/solidus_afterpay/_afterpay_checkout_button.html.erb

```diff
@@ -1,6 +1,6 @@
 <% payment_method = SolidusAfterpay::PaymentMethod.active.first unless local_assigns[:payment_method] %>
 
-<%= render partial: 'solidus_afterpay/afterpay_javascript', locals: { payment_method: payment_method } %>
+<%= render partial: 'solidus_afterpay/afterpay_javascript', locals: { payment_method: payment_method } if Rails.env != "test" %>
```

Comment:
An even better way to do this would be `unless Rails.env.test?`, I think it should work in the same way.